client/src/components/layout/ShowService.js

```diff
@@ -29,30 +34,68 @@ const ShowService = (props) => {
     getServiceAndReviews()
   }, [])
 
-  const reviews = service.reviews.map((review) => {
+  const reviewsMap = service.reviews.map((review) => {
     return(
       <ReviewTile
-        key={review.id}
         review={review}
```

Comment:
I would be surprised if you're not seeing a warning in the browser console when viewing this page....

> Warning: Each child in a list should have a unique "key" prop